exporter/otlphttpexporter/factory_test.go

```diff
@@ -36,6 +36,7 @@ func TestCreateDefaultConfig(t *testing.T) {
 	assert.Equal(t, ocfg.RetryConfig.MaxInterval, 30*time.Second, "default retry MaxInterval")
 	assert.Equal(t, ocfg.QueueConfig.Enabled, true, "default sending queue is enabled")
 	assert.Equal(t, ocfg.Compression, configcompression.Gzip)
+	assert.Equal(t, ocfg.Encoding, EncodingType(""))
```

Comment:
Need to update `factory.CreateDefaultConfig()` to set default encoding to EncodingProto

Reply:
This broke the tests from contrib, and because the default handler is already proto it seemed fine to leave empty

Reply:
I agree that `factory.CreateDefaultConfig()` needs updated to set `Encoding: EncodingProto`.  What broke in contrib?

Reply:
The `f5cloudexporter` broke because one of the tests explicitly creates an instance of the config as part of a test, the new field caused it to mismatch with the asserted one

See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30703

Reply:
The test in f5cloudexporter is fixed, you should be able to update `factory.CreateDefaultConfig()` now